meta/def.rb

```diff
@@ -94,6 +110,7 @@
   mutation 'def foo(_unused = false); end'
   mutation 'def foo(_unused = true); raise; end'
   mutation 'def foo(_unused); end'
+  mutation 'def foo; _unused = true; end'
```

Comment:
> What would be the point in emitting a variable prefixed with an underscore?

I think this is just a badly chosen example that overlaps with the underscore convention.

A mutation from

```
def foo(bar, baz = something)
  body
end
```

to

```
def foo(bar)
  baz = something
  body
end
```

Makes sense. This mutation does not change the semantics of the body, but the arity. It makes sure that there actually is a test call side that does pass in an argument that changes the default behavior.